src/local-history/local-history-output-channel.ts

```diff
@@ -0,0 +1,29 @@
+import * as vscode from 'vscode';
+
+export class OutputManager {
+    private outputChannel: vscode.OutputChannel;
+
+    constructor(outputChannelName: string) {
+        this.outputChannel = vscode.window.createOutputChannel(outputChannelName);
+    }
+
+    /**
+     * Logs the message in the output channel.
+     * @param severity The type of message.
+     * @param timestamp Current time.
+     * @param messages The logged messages.
+     */
+    public appendMessage(severity: OutputChannelSeverity, timestamp: string, messages: string[]): void {
```

Comment:
This could be `private` and we can expose the following methods:

```ts
public appendInfoMessage()
public appendErrorMessage()
public appendWarningMessage()
```

The `severity` will only really require these three types.